client/src/util/catLabelSort.ts

```diff
@@ -11,24 +11,23 @@ TL;DR: sort order is:
 import isNumber from "is-number";
 import * as globals from "../globals";
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-function caseInsensitiveCompare(a: any, b: any) {
+function caseInsensitiveCompare(a: string, b: string): number {
   const textA = String(a).toUpperCase();
   const textB = String(b).toUpperCase();
   return textA < textB ? -1 : textA > textB ? 1 : 0;
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-const catLabelSort = (isUserAnno: boolean, values: any[]): any[] => {
+const catLabelSort = (
+  isUserAnno: boolean,
+  values: Array<string>
+): Array<string> => {
   /* this sort could be memoized for perf */
 
-  const strings: string[] = [];
-  const ints: number[] = [];
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-  const unassignedOrNaN: any = [];
+  const strings = new Array<string>();
```

Comment:
what is the benefit of this syntactic change?   Or is there some change in behavior implied?

Reply:
More readable of a line, is there more of a systematic change then the identifier lookup?

Reply:
Ah I think Airbnb style guide recommends using `[]` over `new Array()`!

https://github.com/airbnb/javascript#arrays

Reply:
Yes, fully agree with airbnb - it is MUCH better IMO to use the JS-style and skip the generic.